pkg/services/auth/auth_token.go

```diff
@@ -201,7 +201,10 @@ func (s *UserAuthTokenService) TryRotateToken(ctx context.Context, token *models
 		return false, nil
 	}
 
-	model := userAuthTokenFromUserToken(token)
+	model, err := userAuthTokenFromUserToken(token)
+	if err != nil {
+		return false, nil
```

Comment:
Here `err` should be returned instead of nil

Reply:
Thanks @papagian!